src/tests/unit/tests/background/feature-flags.test.ts

```diff
@@ -27,6 +27,7 @@ describe('FeatureFlagsTest', () => {
             [FeatureFlags.debugTools]: false,
             [FeatureFlags.exportReportOptions]: false,
             [FeatureFlags.needsReview]: false,
+            [FeatureFlags.saveAndLoadAssessment]: false
```

Comment:
Prettier's complaint about this file is that this new line doesn't end with a comma, which the Prettier vscode extension should have inserted for you. 

Let's make sure you have Prettier set up correctly in vscode. Also, let's verify that `yarn fastpass` on your box detects this issue correctly.